styleguide/placeholder.scss

```diff
@@ -74,3 +83,47 @@
     transition: 50ms fill ease-out;
   }
 }
+
+.kiln-placeholder .placeholder-add-component {
```

Comment:
the rest of the file keeps the placeholder styles separated. this matches that pattern